pkg/agent/cluster/controllers.go

```diff
@@ -31,5 +41,8 @@ func RunControllers() error {
 		return err
 	}
 
+	once.Do(func() {
```

Comment:
This isn't necessary, if somehow `running` gets set to false again the controllers would start but `running` would never be able to get set back to `true` meaning we are back in the same position we are in currently; controllers get restarted on every connect. Just set `running = true`